components/Layout/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react'
+import { Header } from '@/components'
+
+export default function Layout({
+  children
```

Comment:
What's children?

Reply:
Children is generic prop that can accept any type of element from the parent.